dbt_coves/tasks/generate/sources.py

```diff
@@ -278,23 +292,18 @@ def get_nested_keys(self, columns, schema, relation):
                     )
         return result
 
-    def render_templates(self, relation, columns, destination, nested=None):
+    def render_templates(self, relation, data, destination, nested=None):
         context = {
             "relation": relation,
-            "columns": columns,
+            "columns": {},
             "nested": {},
             "adapter_name": self.adapter.__class__.__name__,
         }
-        if nested:
-            context["nested"] = self.get_nested_keys(
-                nested, relation.schema, relation.name
-            )
-            # Removing original column with JSON data
-            new_cols = []
-            for col in columns:
-                if col.name.lower() not in context["nested"]:
-                    new_cols.append(col)
-            context["columns"] = new_cols
+        if type(data) != list:
```

Comment:
Please keep the same signature of nested columns `column key: list of nested items`